### DISCUSSION TITLE
Identity service Error

### DISCUSSION THREAD

**Discussion by gohilamariappan** [Sunbird-RC/community#1039] [General]
Hi Team!

we are getting the below error logs in identity service. we were used the vault service root token .
#########
ERROR Error in checking vault config: AxiosError: Request failed with status code 404.
###########
@holashchand Please suggest.

Regards,
Gohila

**Comment by holashchand** [comment]
Hi @gohilamariappan , the error suggests that vault might not be initialised properly, please check if a key value secrets is created or not. If yes, please verify the correct VAULT_TOKEN is passed or not.

**Comment by gohilamariappan** [comment]
@holashchand we were used the token which generated in the logs .
when we are checking the env variables inside container vault token is there.

Can u please suggest is there any other way to generate token?

Regards,
Gohila

**Reply by holashchand** [reply]
Hi @gohilamariappan, Can you elaborate about how did you setup the vault. we can exec into vault pod and run cmd `vault status` to check if what is initialised or not. Also check here how vault should be setup https://github.com/Sunbird-RC/devops/blob/main/deploy-as-code/helm/v2/registryAndCredentialling/README.md#vault-deployment

If still not solved. let's setup a meeting and share me the link

**Comment by gohilamariappan** [comment]
@holashchand  Thank you so much!

It seems vault is running fine. Issue is in the identity liveness probe configurations..

Regards,
Gohila

**Reply by holashchand** [reply]
Hi @gohilamariappan

Is the issue resolved or do we need to connect and look into?

**Comment by gohilamariappan** [comment]
After changing liveness probe ,It seems running without errors. 
For now no need to connect. will update if anything else required.

Regards,
Gohila
